packages/maps/__tests__/Client.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { Maps as Client} from '../src/Client';
+
+describe('maps', () => {
+  it('can be easily created', () => {
+    const m = new Client('aUser', 'anApiKey');
+    expect(m).toBeTruthy();
+  });
+
+  it('requires credentials', () => {
```

Comment:
Just in case we want to be safer, we should check that it throws when API Key is missing as well.

Reply:
Added a little new test. If we add a Credentials class to ctor, then we will avoid also these duplicities